docs/_layouts/default.html

```diff
@@ -86,7 +86,7 @@
             {% for depr in site.deprecatedVersions %}
             {% if page.url contains depr %}
             <div class="alert alert-danger">
-                  This version is deprecated and will stop working as of Spring 2022. Do not use or upgrade to newest client.
+                  This version is deprecated. As of des 2023 we will only support v16 and higher since we now only support Java >= 11 with jakarta.xml-bind.
```

Comment:
```suggestion
                  This version is deprecated. As of dec 2023 we will only support v16 and higher since we now only support Java >= 11 with jakarta.xml-bind.
```